atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -168,7 +168,7 @@ private SweepResults doRun(TableReference tableRef,
                 .batchSizeHint(batchConfig.candidateBatchSize())
                 .maxTimestampExclusive(sweepTs)
                 .shouldCheckIfLatestValueIsEmpty(sweeper.shouldSweepLastCommitted())
-                .ignoreGarbageCollectionSentinels(!sweeper.shouldAddSentinels())
+                .shouldDeleteGarbageCollectionSentinels(!sweeper.shouldAddSentinels())
```

Comment:
I think this one should be negated?

Reply:
offline discussion: no - the original one is wrong, actually.

- A conservative sweeper should add, but shouldn't delete
- A thorough sweeper shouldn't add and should delete.